batchflow/models/torch/base.py

```diff
@@ -87,7 +87,7 @@ class TorchModel:
         Loss function, might be defined in multiple formats.
 
         If str, then short ``name``.
-        If tuple, then ``(name, *args)``.
+        If tuple, then ``(name, kwargs)``.
```

Comment:
The whole point of tuple was passing args, not kwargs. Otherwise there is no difference with dict.

Reply:
I did not change the behavior of arguments unpacking. I changed docstrings to actual.

Reply:
Now docstrings don't have mention about tuple.
(but it still works)
There is no intersection of our parameter `name` and PyTorch parameters inside. But intersection is in TensorFlow.
It makes sense, to add a new parameter like `graph_name` that is the operation name in a graph. And inside unpacking function for TensorFlow models, replace `graph_name` to `name`.